models/engine/db_storage.py

```diff
@@ -73,4 +73,21 @@ def reload(self):
 
     def close(self):
         """call remove() method on the private session attribute"""
-        self.__session.remove()
+        self.__session.remove(User)
+
+    def get(self, cls, id):
+        """return a count of objects within a class"""
+        if cls is None or id is None:
+            return None
+        obj = self.__session.query().get(id)
+        return obj
```

Comment:
Careful with this, you need to specify what you want to query, this can potentially lead to a lot of errors.

Reply:
[Solved] Included query specifier.